src/components/icons/Icons.jsx

```diff
@@ -182,3 +182,77 @@ export const CalendarIcon = ({ params }) => (
     />
   </svg>
 );
+
```

Comment:
So the idea here is to export all these "icon" svgs from one central location so this is pretty much as generic as we can get it. Eventually would want this to be a generic props paran rather than "params" cause then u can just spread the props and pass anything but haven't gotten around to it yet. Gonna mark as resolved